packages/xcoobee-cookie-kit-react/src/xck-react-theme-overlay.scss

```diff
@@ -25,6 +25,23 @@ $duration: .2s;
   width: 100%;
   transition: opacity 1s ease-out;
   bottom: 0;
+  line-height: 1.5;
+
+  .button,
+  button,
+  input[type="button"],
```

Comment:
Can we remove these `input` selectors? The only `input` elements that I see are check boxes.

Or is there some other markup that is included with the WordPress plugin(s).

This goes for the other rules down below too.

Reply:
@dhurlburtusa We use also `input[type="button"]`. But yes, we should remove the rest I think.